akka-http-core/src/main/java/akka/http/javadsl/model/headers/ContentLength.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.http.javadsl.model.headers;
+
+/**
+ *  Model for the `Content-Length` header.
+ *  Specification: https://tools.ietf.org/html/draft-ietf-httpbis-p1-messaging-26#section-3.3.2
+ */
+public abstract class ContentLength extends akka.http.scaladsl.model.HttpHeader {
+
+    public static ContentLength create(long length) {
```

Comment:
as @2beaucoup mentioned, we don't want people to actually create these – it's a special header that is created/calculated by Akka internally. Please remove the create method :)

https://github.com/akka/akka/pull/20926#issuecomment-231596467